EventsManagementService/EventManagementService.API/Controllers/EventController.cs

```diff
@@ -45,11 +45,11 @@ public async Task<IActionResult> AddEvent([FromBody] EventDTO newEvent)
 
         [Authorize(Roles = "Administrator,Employee,Client")]
         [HttpGet]
-        public async Task<IActionResult> GetAllJobEventsByMonthAndYear([FromQuery] DateTime date)
+        public async Task<IActionResult> GetAllJobEventsByMonthAndYear([FromQuery] int month, int year)

```

Comment:
I think u have to say FromQuery on year as well. It needs to be applied to both.

Reply:
fixed